modules/nf-core/mmseqs/taxonomy/tests/main.nf.test.snap

```diff
@@ -0,0 +1,18 @@
+{
+    "taxonomy - bacteroides_fragilis - genome_nt": {
+        "content": [
+            [
+                "test_query_taxonomy"
+            ]
+        ],
+        "timestamp": "2024-02-23T15:31:53.321266978"
```

Comment:
Interesting that there is no `versions.yml` mentioned although you check for it in main.nf.test. I don't understand that, but apparently the tests pass :shrug: